### ISSUE TITLE
博主你好，train.py在epoch2训练结束后程序运行结束，显示 an illegal memory access was encountered

### ISSUE THREAD

**Issue by Chester818** [bubbliiiing/mask-rcnn-tf2#32]
博主你好，我的运行环境是windows，显卡：RTX A6000 ,    CUDA 11.8，cuDNN 8.7.0, tensorflow-gpu 2.6.0，batch size为1。在训练到epoch2结束后程序运行结束，以下是报错的代码：
2023-03-14 16:05:09.195263: E tensorflow/stream_executor/cuda/cuda_event.cc:29] Error polling for event status: failed to query event: CUDA_ERROR_ILLEGAL_ADDRESS: an illegal memory access was encountered
2023-03-14 16:05:09.196247: F tensorflow/core/common_runtime/device/device_event_mgr.cc:221] Unexpected Event status: 1
请问该如何解决，谢谢博主！
